static/js/composebox_typeahead.js

```diff
@@ -390,6 +390,10 @@ exports.slash_commands = [
         text: i18n.t("/poll Where should we go to lunch today? (Create a poll)"),
         name: "poll",
     },
+    {
+        text: i18n.t("/todo x make a list - with one item (Creates a todo)"),
+        name: "todo",
+    },
```

Comment:
This was added recently. https://github.com/zulip/zulip/pull/17161

Reply:
@YashRE42 Do you mind dropping this commit from your PR?

Reply:
Done.